recipes/pypolyagamma-3/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]
+then
+    export CC=gcc CXX=g++
+fi
+
+if [ $(uname) == Linux ]
+then
+    yum install gsl-devel
```

Comment:
> I tried using conda forge's gsl. I get an error when the code hits #include <gsl/gsl_rng.h>. I ran into the same thing when trying to configure a .travis.yml the other day. The way I fixed it there was to depend on libgsl0-dev but I'm not sure I to do that here.

Could you please share this error? In particular, please add an issue with that info to this [issue tracker]( https://github.com/conda-forge/gsl-feedstock/issues ). Thanks.